### PR TITLE
Add some more doc pointers to the generated code.

### PR DESCRIPTION
**Pull request by thomasvl**

### CODE REVIEW

**Comment by thomasvl** [comment]
@tbkka / @allevato - I believe this is what we kicked around a while back.  Since the docs are updated, seems like it makes sense to add the pointers.

File: Reference/conformance/conformance.pb.swift [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
- * Source: conformance/conformance.proto
- *
- */
+// DO NOT EDIT.
+//
+// Generated by the Swift generator plugin for the protocol buffer compiler.
+// Source: conformance/conformance.proto
+//
+// For information on using the generated types, please see the documenation:
+//   https://github.com/apple/swift-protobuf/blob/master/Documentation/API.md
```
Comment (tbkka) [inline Reference/conformance/conformance.pb.swift]: Minor quibble:  I would just refer to https://github.com/apple/swift-protobuf/ here and let the README point people to more specific documentation.  In particular, these generated files get checked into source control and can live there for years before they're regenerated.

File: Reference/conformance/conformance.pb.swift [hunk 2]
```diff
@@ -140,6 +140,9 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message {
 
   init() {}
 
+  /// Used by the SwiftProtobuf decoding machinery, not generally used directly.
+  /// `init(serializedData:)`, `init(jsonUTF8Data:)`, and other decoding initializers
+  /// are defined in `Message` and `Message+*Additions` files.
```
Comment (tbkka) [inline Reference/conformance/conformance.pb.swift]: I think this comment needs to specifically mention the "SwiftProtobuf library" somewhere.  Perhaps

> Used by the decoding initializers in the SwiftProtobuf library. ...

Or perhaps:

> ... are defined in the SwiftProtobuf library.  See the `Message` and Message+*Additions` files.

File: Reference/conformance/conformance.pb.swift [hunk 3]
```diff
@@ -154,6 +157,9 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message {
     }
   }
 
+  /// Used by the SwiftProtobuf encoding machinery, not generally used directly.
+  /// `Message.serializedData()`, `Message.jsonUTF8Data()`, and other serializer methods
+  /// are defined in `Message` and `Message+*Additions` files.
```
Comment (tbkka) [inline Reference/conformance/conformance.pb.swift]: As above, we should be clear that these "Message and Message+Additions files" are located in the SwiftProtobuf library source code.

File: Reference/conformance/conformance.pb.swift (hunk 1 above)
Comment (thomasvl) [inline Reference/conformance/conformance.pb.swift]: Done.

File: Reference/conformance/conformance.pb.swift (hunk 2 above)
Comment (thomasvl) [inline Reference/conformance/conformance.pb.swift]: Basically used both.

**Pull request merged**